spec/services/checksum_validator_spec.rb

```diff
@@ -203,24 +203,60 @@
         expect(results).to receive(:add_result).with(
           AuditResults::INVALID_MANIFEST, manifest_file_path: 'spec/fixtures/checksum_root01/moab_storage_trunk/xx/444/xx/4444/xx444xx4444/v0001/manifests/signatureCatalog.xml'
         )
-        cv.validate_signature_catalog
+        cv.send(:validate_signature_catalog_listing)
       end
     end
   end
 
-  context '#validate_checksum' do
+  context '#flag_unexpected_data_content_files' do
     let(:druid) { 'bj102hs9687' }
-    let(:cv) { described_class.new(druid, 'fixture_sr1') }
+    let(:cv) { described_class.new(druid, endpoint_name) }
 
-    it 'returns a positive result for a druid that passes validation' do
-      cv.validate_checksum
-      expect(cv.checksum_results.result_array.first).to have_key(:moab_checksum_valid)
+    it 'calls validate_against_signature_catalog on each of the data_content_files' do
+      files = ['spec/fixtures/storage_root01/moab_storage_trunk/bj/102/hs/9687/bj102hs9687/v0001/data/content/eric-smith-dissertation-augmented.pdf',
+               'spec/fixtures/storage_root01/moab_storage_trunk/bj/102/hs/9687/bj102hs9687/v0001/data/content/eric-smith-dissertation.pdf']
+      expect(cv).to receive(:data_content_files).and_return(files)
+      files.each do |file|
+        expect(cv).to receive(:validate_against_signature_catalog).with(file)
+      end
+      cv.send(:flag_unexpected_data_content_files)
     end
 
-    it 'returns error codes for a druid that fails validation' do
-      cv = described_class.new(druid, endpoint_name)
-      cv.validate_checksum
-      expect(cv.checksum_results.result_array.first).to have_key(:file_not_in_manifest)
+    it 'calls AuditResults.report_results' do
```

Comment:
I believe this shouldn't happen here but in #validate_checksum, no?

Also, not your "fault" but `validate_checksums` is more accurate as we don't validate only ONE.